src/glog/logging.h.in

```diff
@@ -135,50 +135,6 @@ typedef unsigned __int64 uint64;
 
 @ac_google_end_namespace@
 
-#ifdef GLOG_CUSTOM_PREFIX_SUPPORT
```

Comment:
Instead of rearraning the code, you can declare the flags above the `struct` definition.